[2021-08-10 18:55:52,188] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Covid.Covidtask2 2021-08-10T13:10:13.395265+00:00 [queued]>
[2021-08-10 18:55:52,197] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Covid.Covidtask2 2021-08-10T13:10:13.395265+00:00 [queued]>
[2021-08-10 18:55:52,197] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-08-10 18:55:52,197] {taskinstance.py:1068} INFO - Starting attempt 1 of 8
[2021-08-10 18:55:52,197] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-08-10 18:55:53,130] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): Covidtask2> on 2021-08-10T13:10:13.395265+00:00
[2021-08-10 18:55:53,135] {standard_task_runner.py:52} INFO - Started process 33842 to run task
[2021-08-10 18:55:53,142] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'Covid', 'Covidtask2', '2021-08-10T13:10:13.395265+00:00', '--job-id', '357', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmph61qt5zv', '--error-file', '/tmp/tmppv512z4e']
[2021-08-10 18:55:53,143] {standard_task_runner.py:77} INFO - Job 357: Subtask Covidtask2
[2021-08-10 18:55:54,657] {logging_mixin.py:104} INFO - Running <TaskInstance: Covid.Covidtask2 2021-08-10T13:10:13.395265+00:00 [running]> on host deepika-Inspiron-5570
[2021-08-10 18:55:54,879] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=sthadpka93@gmail.com
AIRFLOW_CTX_DAG_OWNER=postgres
AIRFLOW_CTX_DAG_ID=Covid
AIRFLOW_CTX_TASK_ID=Covidtask2
AIRFLOW_CTX_EXECUTION_DATE=2021-08-10T13:10:13.395265+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-10T13:10:13.395265+00:00
[2021-08-10 18:56:01,553] {python.py:151} INFO - Done. Returned value was: None
[2021-08-10 18:56:01,556] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=Covid, task_id=Covidtask2, execution_date=20210810T131013, start_date=20210810T131052, end_date=20210810T131101
[2021-08-10 18:56:01,761] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-10 18:56:01,805] {local_task_job.py:151} INFO - Task exited with return code 0
